synapse/module_api/__init__.py

```diff
@@ -759,6 +790,8 @@ def generate_short_term_login_token(
 
         Added in Synapse v1.9.0.
 
+        This is deprecated in favor of create_login_token.
```

Comment:
Can you add information about the version in which the deprecation happened please?

Reply:
Done 👍